docs/application/dotnet/api/overview.md

```diff
@@ -24,6 +24,8 @@ The Tizen.NET contains a few limitations on the Xamarin.Forms APIs; these limita
 
 A list of limitations is available at [Current Xamarin.Forms limitations](xamarin.forms-limitations.html).
 
+You can efficiently add Tizen wearable device specific user interfaces. Extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/).
```

Comment:
Extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/). 

Please change as:

The extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/). 

Suggestion:
When I clicked on "https://github.com/Samsung/Tizen.CircularUI/" link it is taking me to Samsung/Tizen.CircularUI github site.

Can we change Samsung/Tizen.CircularUI  to Samsung/Tizen.Wearable.CircularUI  to avoid confusion?